### DISCUSSION TITLE
Column selection confusing

### DISCUSSION THREAD

**Discussion by moritzvolkmar** [Samedis-care/samedis-care-community#87] [Shipped - Live on production]
The column selection is currently just a list of all the fields available for selection. 
As the terms are neither grouped nor sorted alphabetically, you have to search for a very long time to find the right information.

![Feature Spalten gruppieren](https://github.com/Samedis-care/samedis-care-community/assets/80519295/fdd4d284-accc-44d2-9f53-ec78e14d6470)

Maybe you can optimize something here.

**Comment by YvesR** [comment]
Next update we will sort alphabetically and modify design for better reading.
